modules/plugin/coverage-multidim/coverage-api/pom.xml

```diff
@@ -93,6 +93,15 @@
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
Why is JAXB needed here?

Reply:
Not sure, but it fails to build on my machine without it (as at least a test dependency):

```
Running org.geotools.imageio.GeoSpatialImageReaderTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.003 s <<< FAILURE! - in org.geotools.imageio.GeoSpatialImageReaderTest
[ERROR] testReader(org.geotools.imageio.GeoSpatialImageReaderTest)  Time elapsed: 0.002 s  <<< ERROR!
java.lang.IllegalArgumentException: java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at org.geotools.imageio.GeoSpatialImageReaderTest.testReader(GeoSpatialImageReaderTest.java:102)
Caused by: java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at org.geotools.imageio.GeoSpatialImageReaderTest.testReader(GeoSpatialImageReaderTest.java:102)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
	at org.geotools.imageio.GeoSpatialImageReaderTest.testReader(GeoSpatialImageReaderTest.java:102)
```

Reply:
there is another commit coming (maybe) that centralizes the version and makes it a test dependency

Reply:
So the jaxb dependency should be coming from imagemosaic, which is a dependency of coverage-api. But repeating it does not hurt.